packages/kbn-build/src/commands/run.ts

```diff
@@ -1,13 +1,18 @@
 import chalk from 'chalk';
 
-import { topologicallyBatchProjects } from '../utils/projects';
+import { Project } from '../utils/project';
+import { ProjectGraph, topologicallyBatchProjects } from '../utils/projects';
 import { parallelizeBatches } from '../utils/parallelize';
 
 export const name = 'run';
 export const description =
   'Run script defined in package.json in each package that contains that script.';
 
-export async function run(projects, projectGraph, { extraArgs }) {
+export async function run(
+  projects: Map<string, Project>,
```

Comment:
Maybe create a type that's used both in the commands and in `packages/kbn-build/src/commands/index.ts`?

Reply:
(Could use generics for the `options` arg)